### PR TITLE
ci(fix): check that no files were changed when performing a lint

### PR DESCRIPTION
**Pull request by zimeg**
### Type of change

- [x] Bug fix

### Summary

This PR checks if the CI lintings catch unexpected formatting or similar strangness with `biome` 🔍  Planning to share findings in comments below 🙏 

### Requirements

- [x] I’ve checked my submission against the Samples Checklist to ensure it complies with all standards
- [x] I have ensured the changes I am contributing align with existing patterns and have tested and linted my code
- [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct)

### CODE REVIEW

**Comment by zimeg** [comment]
https://github.com/slack-samples/bolt-ts-starter-template/actions/runs/11675826478/job/32511044471?pr=131#step:5:17

```
> bolt-ts-starter-template@1.0.0 test
> npm run build && npm run lint

...

> bolt-ts-starter-template@1.0.0 lint
> npx @biomejs/biome check --write *.ts listeners

Checked 15 files in 8ms. Fixed 1 file.
```

The fixed file here unfort. isn't added to the PR though'twould be neat 👀  Will follow up with proposed changes!

File: package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "build": "tsc",
     "start": "npm run build && node ./dist/app.js",
-    "lint": "npx @biomejs/biome check --write *.ts listeners",
+    "lint": "npx @biomejs/biome check *.ts listeners",
+    "lint:fix": "npx @biomejs/biome check --write *.ts listeners",
```
Comment (zimeg) [inline package.json:10]: No reference to `npm run lint` in the `README` which I think is alright, but I'm unsure if the `:` approach makes sense here 🤔

Comment (seratch) [reply package.json:10]: Adding `npm run lint:ci` and using it for CI jobs may be another option, but I'm happy to let @filmaj and you decide the naming tomorrow!

Comment (filmaj) [reply package.json:10]: Either works for me 👍

**Review by filmaj — APPROVED** [review]

**Comment by filmaj** [comment]
A potential related option is to consider running the `--write` run script in a pre-commit hook. I even opened an issue for this in the node-slack-sdk: https://github.com/slackapi/node-slack-sdk/issues/2034

File: package.json (hunk 1 above)
Comment (zimeg) [reply package.json:10]: > Adding `npm run lint:ci` and using it for CI jobs may be another option

@seratch Thank you for the suggestion and apologies for a slow response 😓

I like `lint:ci` but when making that change I found an update to `npm run test` which was somewhat unexpected when testing other code changes.

I do think an `npm run ci` job might be neat, but right now we're using `npm run test` to check `build` and `lint` which I think covers this 👀

**Comment by zimeg** [comment]
@seratch @filmaj Thank y'all both for the reviews!

https://github.com/slackapi/node-slack-sdk/issues/2034 is quite interesting - I haven't seen this before, but it'd be neat to have these checks before a `git push`. I find myself testing with CI sometimes 😅

Going to merge this now and follow up with the same change elsewhere 🚢

**Pull request merged**